var/spack/repos/builtin/packages/sepp/package.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Sepp(Package):
+    """SEPP stands for SATe-enabled phylogenetic placement"""
+
+    homepage = "https://github.com/smirarab/sepp"
+
+    url = "https://github.com/smirarab/sepp/archive/refs/tags/4.5.1.tar.gz"
+    maintainers = ["snehring"]
+
+    version("4.5.1", sha256="51e052569ae89f586a1a94c804f09fe1b7910a3ffff7664e2005f18c7d3f717b")
+
+    depends_on("python@3.7:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+
+    depends_on("java@1.8:", type="run")
+    depends_on("pasta@1:", type="run")
+    depends_on("blast-plus@2.10.1:", type="run")
+
+    def install(self, spec, prefix):
+        python = which("python")
+        install_tree(".", prefix)
+        with working_dir(prefix):
+            python("setup.py", "config", "-c")
+            python("setup.py", "install")
+            python("setup.py", "upp", "-c")
```

Comment:
Instead of `-c` is it possible to use `--prefix=<prefix>`? I would like to avoid copying all the random text files from the root of the project if possible.

Reply:
Without the -c it'll write a config file in ~/.sepp/ for the installing user. I can see if it still works as expected with busco though.

Reply:
Ah I thought `-c` simply meant install locally instead of to the Python installation prefix. Maybe combine `-c` and `--prefix` then?

Reply:
I think it's picking up the prefix correctly as is, in that it is making a busco in the site-packages dir as expected for the package prefix. All -c seems to do is control where the 'config' lives, either in the user home or in the directory setup.py is run from. I can add it in explicitly though.

Reply:
Ugh ignore above about busco, other package obviously. The -c stuff holds true though, and omitting it it causes issues for people other than the installing user.

Reply:
The config path situation for sepp is messy. Let me see if I can improve it any.

Reply:
Not copying the entire tree doesn't seem to be an option. Not without changes to how the setup.py works. It hard codes a lot of paths based on where it's executed which then get passed into the config.

Reply:
That's unfortunate. Could we delete everything that isn't `prefix.bin` or `prefix.lib` so that there aren't a ton of README files in the installation prefix? Those will likely be ugly once you try to create a single symlinked view.

Reply:
Yeah, I'll see if I can pare it down from the whole tree to a specific set of files.

Reply:
Much cleaner result now.